src/options.c

```diff
@@ -81,6 +83,51 @@ static int parse_ipv4(struct super_block *sb, char *str,
 	return 0;
 }
 
+static int parse_bdev_path(struct super_block *sb, substring_t *substr,
+			      char **bdev_path_ret)
+{
+	char *bdev_path;
+	struct inode *bdev_inode;
+	struct path path;
+	bool got_path = false;
+	int ret;
+
+	bdev_path = match_strdup(substr);
+	if (!bdev_path) {
+		scoutfs_err(sb, "bdev string dup failed");
+		ret = -ENOMEM;
+		goto out;
+	}
+
+	ret = kern_path(bdev_path, LOOKUP_FOLLOW, &path);
+	if (ret) {
+		scoutfs_err(sb, "path %s not found for bdev: error %d",
+			    bdev_path, ret);
+		goto out;
+	}
+
```

Comment:
I just noticed that got_path is never set, we'd set it here?

Reply:
whups, fixed.